src/examples/real.lean

```diff
@@ -14,8 +19,14 @@ open liquid_tensor_experiment category_theory category_theory.limits opposite
 
 example : conditionally_complete_linear_order ℝ := infer_instance
 example : linear_ordered_field ℝ := infer_instance
+example : conditionally_complete_linear_ordered_field ℝ := infer_instance
```

Comment:
I think we can just remove the two lines above this.

Reply:
Done